### PR TITLE
KK-1137 | feat: markdown additional notes for child detail

### PR DESCRIPTION
**Pull request by melniiv**
## Description

Notes field with markdown editor should be shown in the child detail page under past event. Notes has view and edit mode. view mode is displayed as a card as any past events. When clicking edit button, markdown editor should be displayed and used can enter past events or any other additional info:

Edit mode with sanitising user input:
image

Preview mode:
 
image

## Context

[KK-1137](https://helsinkisolutionoffice.atlassian.net/browse/KK-1137)

## Screenshots

[KK-1137]: https://helsinkisolutionoffice.atlassian.net/browse/KK-1137?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://kukkuu-local.api.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr559.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-ui-pr559.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-local.api.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr559.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-ui-pr559.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-local.api.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-local.api.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr559.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://kukkuu-ui-pr559.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr559.dev.hel.ninja** :rocket::rocket::rocket:

File: src/domain/child/queries/ChildQueries.ts [hunk 1]
```diff
@@ -183,3 +183,17 @@ export const childByIdQuery = gql`
     }
   }
 `;
+
+export const childNotesByIdQuery = gql`
+  fragment ChildNotesByIdQueryFields on ChildNotesNode {
+    id
+    childId
+    notes
+  }
```
[the remark this answers is not in the document]
Comment (melniiv) [reply src/domain/child/queries/ChildQueries.ts]: fixed

**Comment by terovirtanen** [comment]
**KUKKUU-UI branch is deployed to platta: https://kukkuu-ui-pr559.dev.hel.ninja** :rocket::rocket::rocket:

**Review by karisal-anders — APPROVED** [review]
The clarity of the `Card` component with the combination of `action`, `primaryAction`, `withCardClickAction` and `withAction` does suffer a bit, so that could possibly be refactored for clarity, not required for merging though IMO

**Pull request merged**